virt/virtual_machines/virt-accessing-vm-consoles.adoc

```diff
@@ -30,6 +30,11 @@ include::modules/virt-temporary-token-VNC.adoc[leveloffset=+2]
 [id="serial-console_virt-accessing-vm-consoles"]
 == Connecting to the serial console
 
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../virt/about_virt/virt-architecture#virt-about-ssp-operator_virt-architecture[About the Scheduling, Scale, and Performance (SSP) Operator]
```

Comment:
After the next section's heading seems like the wrong place to put Additional resources. I suggest moving and inserting these lines before line 29.